docs/sources/explore/metrics-explore.md

```diff
@@ -0,0 +1,72 @@
+---
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Grafana Metrics Explore
+aliases: 
+description: This topic describes the Metrics Explore feature
+weight: 200
+---
+
+# Grafana Metrics Explore
```

Comment:
@mary-sitzenstatter can you confirm that this should be "Grafana Explore Metrics" throughout, vs. "... Metrics Explore"

Reply:
I would use "Metrics Explore"!

Reply:
We need to come to a decision on the name of this feature

Reply:
done - explore metrics and not metrics explore